common/src/main/resources/startup.properties

```diff
@@ -243,6 +243,9 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 # Authorization Enabled flag: false (default)|true
 *.falcon.security.authorization.enabled=false
 
+# CSRF filter enabled flag: false (default) | true
+*.falcon.security.csrf.enabled=false
```

Comment:
It will be a good idea to mention the dependency on other properties in order to enable this property.